video/out/opengl/egl_helpers.c

```diff
@@ -193,7 +193,11 @@ bool mpegl_create_context_cb(struct ra_ctx *ctx, EGLDisplay display,
     MP_VERBOSE(ctx, "EGL_VERSION=%s\nEGL_VENDOR=%s\nEGL_CLIENT_APIS=%s\n",
                STR_OR_ERR(version), STR_OR_ERR(vendor), STR_OR_ERR(apis));
 
+#if HAVE_ANDROID
+    int es[] = {3, 2, 0}; // preference order
+#else
```

Comment:
Some Android TV devices offer desktop opengl, but that doesn't work with the GLES GL_TEXTURE_EXTERNAL_OES extension. I had to switch the preference order here to make the hwdec work.

Reply:
I think an alternative might be --opengl-es=yes